README.md

```diff
@@ -13,4 +13,11 @@ This is a library for serialization & deserialization of data structures used in
 
 ## Documentation
 
-You can find documentation [here](https://docs.cardano.org/projects/cardano-serialization-lib/en/latest/)
\ No newline at end of file
+You can find documentation [here](https://docs.cardano.org/projects/cardano-serialization-lib/en/latest/)
+
+
+
+### Credits
```

Comment:
I don't think this is a legally accepted way to credit their work. I think we have to do the following:

1) Create a new file called `LICENSE-IOHK` at the root of our project
2) Put in the content of https://github.com/input-output-hk/chain-libs/blob/master/LICENSE-MIT as-is in this new file
3) In our repository's LICENSE file, add the following at the bottom of the file

```
Includes other software related under the MIT license:
- chain-libs, Copyright 2018-2019 IOHK. For licensing see /LICENSE-IOHK
```